api/tests/unit/domain/models/TargetedTube_test.js

```diff
@@ -27,4 +27,32 @@ describe('Unit | Domain | Models | Target-Profile/TargetedTube', () => {
       expect(isIncluded).to.be.false;
     });
   });
+
+  describe('get# skillIds', () => {
+
+    it('should return list of skill ids', () => {
+      // given
+      const skill1 = domainBuilder.buildTargetedSkill({ id: 'skillId1', tubeId: 'tubeId' });
+      const skill2 = domainBuilder.buildTargetedSkill({ id: 'skillId2', tubeId: 'tubeId' });
+      const tube = domainBuilder.buildTargetedTube({ id: 'tubeId', skills: [skill1, skill2] });
+
+      // when
+      const skillIds = tube.skillIds;
+
+      // then
+      expect(skillIds).to.deep.include.members(['skillId1', 'skillId2']);
+      expect(skillIds).to.have.length(2);
+    });
+
+    it('should return an empty array when tube has no skills', () => {
+      // given
+      const tube = domainBuilder.buildTargetedTube({ id: 'tubeId', skills: [] });
+
+      // when
+      const skillIds = tube.skillIds;
+
+      // then
+      expect(skillIds).to.deep.equal([]);
```

Comment:
```suggestion
      expect(answers).to.be.empty;
```

Reply:
je n'étais pas sûre que to.be.empty me garantissait le type (à savoir array vide)